src/test/java/bio/terra/service/duos/DuosDaoTest.java

```diff
@@ -33,21 +37,24 @@
 @EmbeddedDatabaseTest
 public class DuosDaoTest {
 
-  @Autowired private DuosDao duosDao;
+  @Autowired private NamedParameterJdbcTemplate jdbcTemplate;
+  private DuosDao duosDao;
+
+  private static final String TDR_SERVICE_ACCOUNT_EMAIL = "tdr-sa@a.com";
 
   private UUID duosFirecloudGroupId;
   private String duosId;
   private String firecloudGroupName;
   private String firecloudGroupEmail;
-  private String tdrServiceAccountEmail;
   private DuosFirecloudGroupModel toInsert;
 
   @Before
   public void before() {
+    duosDao = new DuosDao(jdbcTemplate, TDR_SERVICE_ACCOUNT_EMAIL);
```

Comment:
If you wanted to work around this need to construct the Dao class by hand, you could convert the service account email to a record type (instead of a qualified String) and then mock it in this class:
```
  @MockBean private ServiceAccountEmail serviceAccountEmail;
```
Using a record type would let you get rid of all the `@Qualified` attributes too, if that's something you've been thinking about doing